api/rest-client.ts

```diff
@@ -1,9 +1,24 @@
-import axios, { AxiosRequestConfig, ResponseType } from 'axios'
+import https from 'https'
+import got, { NormalizedOptions, Options, Headers } from 'got'
+import CacheableLookup from 'cacheable-lookup'
 import { delay, getHardwareId, logError, logInfo, stringify } from './util'
 import * as querystring from 'querystring'
 import { AuthTokenResponse, SessionResponse } from './ring-types'
 import { ReplaySubject } from 'rxjs'
 
+const cacheableLookup = new CacheableLookup();
+const httpsAgent = new https.Agent({ keepAlive: true });
+https.globalAgent = httpsAgent;
```

Comment:
Shouldn't need to set this anymore.  Setting `globalAgent` seems like it could have an impact on other homebridge modules as well, so is something I wanted to avoid.  I'd prefer to use `keepaliveagent`, as shown here: https://www.npmjs.com/package/got#agent

Reply:
True that!